### PR TITLE
Add important information about alternative text

### PR DESCRIPTION
**Pull request by ssimo3lsuhsc**
I have an assignment that is due 2018-11-02 23:00:00GMT+10.5. I need to document that CKEditor 4's default image plugin supports adding alternative text. The only other person who will ever read my assignment is Dr. Scott Hollier, a web accessibility expert who himself is blind. For him, it is important that the phrase "alternative text" be in the documentation of CKEditor 4's default image plugin. And that is what I have done.

### CODE REVIEW

**Comment by ssimo3lsuhsc** [comment]
I must clarify that [Dr. Hollier has disclosed himself that he is clinically blind](http://www.hollier.info/about/).

**Comment by AnnaTomanek** [comment]
Thank you for your contribution. However, I don't think this sort of "assignment" can be completed as you planned, for a couple of reasons. Firstly, the official documentation is updated online together with CKEditor 4 releases so there is no chance it will be updated before Nov 2, 2018. Secondly, your suggestion was added in a wrong place - this section explains the differences between our three image plugins ("It provides the following features when compared to two alternative image plugins"), and all three provide the ability to insert alternative text. This is like accessibility ABC, CKEditor is compliant with WCAG 2.0 and Section 508, so it's a must.

If I may advise something, I'd suggest to add information about image `alt` here: https://ckeditor.com/docs/ckeditor4/latest/guide/dev_a11y.html#visual-impairment

**Comment by ssimo3lsuhsc** [comment]
Thanks, Anna. However, I have identified an important accessibility standard with which CKEditor has been noncompliant for **more than three years.** Does the acronym "WTAG" ring a bell? Should I create a new issue?

File: docs/guide/dev/a11y/README.md [hunk 1]
```diff
@@ -134,6 +134,10 @@ Color blind users may have trouble accessing information that is conveyed with c
 
 The following features can be used by developers to ensure that content created with CKEditor is accessible.
 
+#### Support for the `alt` Attribute in Images
+
+All three of CKEditor 4's Image plugins offer support for the `alt` attribute. This is done through an "Alternative Text" prompt. If the prompt is left blank, the `alt` attribute will be `""`.
```
Comment (AnnaTomanek) [inline docs/guide/dev/a11y/README.md]:
```suggestion
All official CKEditor 4 image plugins offer support for adding alternative text through the `alt` attribute. This is achieved by using the "Alternative Text" field in the image dialog. If the field is left blank, the `alt` attribute will be empty (`""`).
```

**Comment by AnnaTomanek** [comment]
Thank you. Please report issues related to CKEditor code and compatibility directly in the [CKEditor repository](https://github.com/ckeditor/ckeditor-dev/issues).

**Comment by f1ames** [comment]
Hi,
Thank you for supporting Open Source and contributing to the CKEditor 4 project!
It's been a while since we last heard from you. We're closing this PR for now. Still, feel free to provide us with the requested feedback so that we can reopen this PR. Thank you!

**Pull request closed without merging**